run.bat

```diff
@@ -1,2 +1,2 @@
 @echo off
-.\venv\Scripts\activate && waitress-serve --listen=*:8000 --threads=10 --channel-timeout=1800 MobSF.wsgi:application
+.\venv\Scripts\activate && waitress-serve --listen=127.0.0.1:8000 --threads=10 --channel-timeout=1800 MobSF.wsgi:application

```

Comment:
Keep `--listen` be default as *:8000. Allow user to set `ip:port` as first argument for localhost and document it as default in docs.